setup.py

```diff
@@ -31,8 +40,8 @@
         'Natural Language :: Russian',
         'Programming Language :: Python',
         'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.6',
         'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
```

Comment:
It'd be nice to add Python 3.5 classifier - Python 3.2 is also a Python 3, but it doesn't look supported